### ISSUE TITLE
PROPOSAL: New list UG_JINJA-CBO code verification

### ISSUE THREAD

**Issue by andreaszenasidi** [org-id/register#465]
### Proposal

I am reaching out to verify that the _UG_JINJA-CBO_ code is the correct one to use for Community Based Organisations (CBO)  in Uganda. The reasoning behind creating a district level registrar is that the publisher will not register with the National Bureau of NGOs (http://org-id.guide/list/UG-NGO) because the registration process is too long and they only operate in Jinja district hence they aren't required to register at the national level.
A publisher needs to use this code as soon as possible. I will prepare and submit the new list to org-id shortly.

**The name of the list (or the organization that manages the list)**
Jinja District Non-Governmental Monitoring Committee

**A suggested code for the list (optional)**
UG_JINJA-CBO

**A short description of the list**
Community Based Organisations operating in Jinja District are registered at the district level by the District NGO Monitoring Committee (DNMC). 

**The country or countries that the list covers**
Uganda, Jinja District

**A URL for information on the list, and a URL for looking up identifiers (if available)**
https://www.ngobureau.go.ug/en/services/community-based-organisations
https://wikiprocedure.com/index.php/Uganda_-_Register_for_Community_Based_Organisation_CBO

**Example of a certificate**
![image](https://user-images.githubusercontent.com/74558657/131845576-4d90abe8-e9b9-4deb-b1c8-b77590ddcd13.png)

**Comment by kd-ods** [comment]
Hi @andreaszenasidi -

Given the subnational nature of this list, I think the list name code should use the [ISO 3166-2 subdivision code](https://en.wikipedia.org/wiki/ISO_3166-2:UG] for brevity. So the full code fort the list would be **UG-204CBO**. ('Jinja' should be used in the name/title of the list for clarity.)

Before proceeding, can I just check: are you intending to use the 'Reg No' on the certificate as the organisation's ID? If so, are you certain that it is stable across time and doesn't change when the organisation renews its permit?

**Comment by andreaszenasidi** [comment]
@kd-ods Yes, I wanted to use the Reg No as the org ID. The NGO said that the certificate is issued only once. They also have a permit (please see screengrab below) which is renewed annually. 
![image](https://user-images.githubusercontent.com/74558657/134913258-6872c3ca-c090-4b86-9a1a-7654b1d8a716.png)

**Comment by kd-ods** [comment]
@andreaszenasidi - thanks for confirming. Please go ahead and submit a PR for the list UG-204CBO.
